packages/test/test-end-to-end-tests/src/test/summarizeIncrementally.spec.ts

```diff
@@ -0,0 +1,245 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
```

Comment:
>import [](http://example.com/codeflow?start=0&length=6)

BTW, now that you have a clear classification of tests between these two modules; consider adding an explanatory annotation blob at the top of each file. This helps ensure other developers understand the classification when adding new tests in this area. 

I noticed we don't have such annotations for most of our test files, but it can be immensely helpful in the long run.

Reply:
Good point. I do write a description of the test just before the describe block (see any of the GC end-to-end tests for an example). I missed adding one in this case!

Reply:
Added.